src/server/api/organization.js

```diff
@@ -21,8 +21,16 @@ export const resolvers = {
       await accessRequired(user, organization.id, "SUPERVOLUNTEER");
       return getCampaigns(organization.id, cursor, campaignsFilter, sortBy);
     },
+    campaignsCount: async (organization, _, { user }) => {
+      await accessRequired(user, organization.id, "OWNER", true);
+      return getCampaignsCount(organization.id);
```

Comment:
`return r.getCount(r.knex("campaign").where({"organization_id": organization.id, is_archived: false}))` is simpler than the indirection with buildCampaignQuery, but if you have use cases that will grow then this is fine.

Reply:
Yeah that seems better - I'll update!